questions.md

```diff
@@ -444,6 +444,22 @@ without going a deep philosophical discussion.
 
 
 
+```
+
+## 15. When and where did you heard first time of the the word *open source software* ? 
```

Comment:
@ssurbhi560 we're _stoked_ for your enthusiasm and wanting to add questions. 🎉 
The _general_ concept of Open Source is not strictly relevant to @dwyl ... 💭